src/components/NewCommentForm.tsx

```diff
@@ -1,13 +1,16 @@
 import classNames from 'classnames';
 import React, { useState } from 'react';
-import { CommentData } from '../types/Comment';
+import { useAppDispatch } from '../app/hooks';
+import { addCommentAsync } from '../features/commentsSlice';
+import { Post } from '../types/Post';
 
 type Props = {
-  onSubmit: (data: CommentData) => Promise<void>;
+  post: Post | null;
 };
 
-export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
+export const NewCommentForm: React.FC<Props> = ({ post }) => {
   const [submitting, setSubmitting] = useState(false);
```

Comment:
add `is` to variable naming for boolean types. Fix all boolean variables according tips ->
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig